sources/packages/backend/libs/queue/src/utilities/queue.util.ts

```diff
@@ -0,0 +1,13 @@
+import { BullModuleOptions } from "@nestjs/bull";
+import { Queues, QUEUE_PREFIX } from "../constants/queue.constant";
+
+/**
+ * Builds the bull module options to register queues in a dynamic wau.
+ * @returns bull module options with all existing queues.
+ */
+export const getQueueModules = (): BullModuleOptions[] => {
```

Comment:
`getQueueModules` does not sound accurate here.
I am actually confused here, the method is apparently returning `BullModuleOptions` but the prefix is a property from `QueueOptions`, right? Is it used for 2 purposes?
For instance, I believe that `QueueOptions` has more options like `limiter` that I am not aware which is the default value but I believe that we need to figure out what would be the best config.